Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoISE.yaml

```diff
@@ -0,0 +1,70 @@
+Parser:
+  Title: Authentication ASIM parser for Cisco ISE
+  Version: '1.0'
+  LastUpdated: July 19, 2023
+Product:
+  Name: Cisco ISE
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Cisco ISE Security Events 
+  Link: https://www.cisco.com/c/en/us/td/docs/security/ise/3-2/admin_guide/b_ise_admin_3_2/b_ISE_admin_32_maintain_monitor.html#ID58
+Description: |
+  This ASIM parser supports normalizing Cisco ISE events produced by the Microsoft Sentinel Cisco ISE connector to the ASIM Authentication schema.
+ParserName: ASimAuthenticationCiscoISE
+EquivalentBuiltInParser: _ASim_Authentication_CiscoISE
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
```

Comment:
@spsocprime  Team all changes looks good. Please format query to organize code

Reply:
@spsocprime Can you please take a look?

Reply:
done